### PR TITLE
Comment Management

### PR DESCRIPTION
**Pull request by cpuwhiz11**
Delete or keep comments that have been flagged for content. Should satisfy issue #81

### CODE REVIEW

File: frontend/aiagallery/source/class/aiagallery/module/dgallery/appinfo/Comment.js [hunk 1]
```diff
@@ -12,9 +12,41 @@ qx.Class.define("aiagallery.module.dgallery.appinfo.Comment",
   implement : [qx.ui.form.IModel],
   include   : [qx.ui.form.MModelProperty],
   
-  construct : function(data)
+ /**
+  * Create a new comment object. 
+  *
+  * @param data {Map}
```
Comment (cpuwhiz11) [inline frontend/aiagallery/source/class/aiagallery/module/dgallery/appinfo/Comment.js:18]: At least I think that is what this is for, not sure it is ever used.

**Comment by derrell** [comment]
Testing this... If I flag a comment and then view the Comment Management page, I see no flagged comments displayed...???

**Comment by cpuwhiz11** [comment]
Flagged comments will only show up in the management page if they have received 5 or more flags (MAX_FLAGGED in Constants.js). Just one flag is not enough.

**Comment by derrell** [comment]
I think that's probably not adequate for an administrator's view. An administrator may want to watch for even a very few flagged comments, perhaps when there are a set of users causing problems. I believe we should show all apps with flagged comments in the comment management page.

**Pull request merged**